packages/plugins/plugin-space/src/types/thread.ts

```diff
@@ -34,9 +34,18 @@ export const ActorSchema = S.mutable(
 
 export type ActorType = S.Schema.Type<typeof ActorSchema>;
 
+export enum MessageState {
+  NONE = 0,
+  ARCHIVED = 1,
+  DELETED = 2,
+  SPAM = 3,
+}
+
 export class MessageType extends TypedObject({ typename: 'dxos.org/type/Message', version: '0.1.0' })({
   /** ISO date string when the message was sent. */
   timestamp: S.String,
+  /** Message state. */
+  state: S.optional(S.Enums(MessageState)),
```

Comment:
this is what `properties` is for, we should reserve top-level fields for generic things

Reply:
i think this is generic. all messages should have a state.

Reply:
I don't understand how these states make sense outside of the context of syncing w/ an external email service. Let's discuss